scipy/optimize/_differentialevolution.py

```diff
@@ -254,6 +255,13 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         ``'vectorized'``, and when to use ``'workers'``.
 
         .. versionadded:: 1.9.0
+        
+    strategy_func : fn(candidate, population, rng=None)
```

Comment:
> BTW, I'm not sure if we'll have a separate keyword argument or include it as part of the `strategy` keyword. pinging @mdhaber for a second opinion.

Not a bad idea - I don't see any issues with implementing it that way, since "strategy" is not used when using strategy_func.